### PR TITLE
SCB-783 Update the License Files

### PR DESCRIPTION
**Pull request by asifdxtreme**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: saga-distribution/src/release/LICENSE [hunk 1]
```diff
@@ -347,42 +304,37 @@ Netty/Handler/Proxy (http://netty.io/netty-handler-proxy/) io.netty:netty-handle
...
-Spring Boot AutoConfigure (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-autoconfigure:jar:1.5.14.REALEASE
-Spring Boot JDBC Starter (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-starter-jdbc:jar:1.5.14.REALEASE
-Spring Boot Logging Starter (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-starter-logging:jar:1.5.14.REALEASE
-Spring Boot Starter (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-starter:jar:1.5.14.REALEASE
-Spring Boot Tomcat Starter (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-starter-tomcat:jar:1.5.14.REALEASE
-Spring Boot Web Starter (http://projects.spring.io/spring-boot/) org.springframework.boot:spring-boot-starter-web:jar:1.5.14.REALEASE
-Spring Context (https://github.com/spring-projects/spring-framework) org.springframework:spring-context:jar:4.3.18.RELEASE
-Spring Core (https://github.com/spring-projects/spring-framework) org.springframework:spring-core:jar:4.3.18.RELEASE
+SnakeYAML (http://www.snakeyaml.org) org.yaml:snakeyaml:bundle:1.1x
```
Comment (WillemJiang) [inline saga-distribution/src/release/LICENSE]: We need to keep the specific version as the License could be changed between different version.

Comment (asifdxtreme) [reply saga-distribution/src/release/LICENSE]: okay I will revert back the changes

File: saga-distribution/src/release/LICENSE [hunk 2]
```diff
@@ -245,14 +236,6 @@ PostgreSQL license.
 For details, see https://github.com/pgjdbc/pgjdbc-parent-poms
 You can find a copy of the License at licenses/LICENSE-postgresql
 
-===========================================================================
```
Comment (WillemJiang) [inline saga-distribution/src/release/LICENSE]: We need to keep the Json bundle.

**Comment by asifdxtreme** [comment]
@WillemJiang I have updated the PR with the changes

**Comment by asifdxtreme** [comment]
I will cross check with the bin distribution

**Comment by asifdxtreme** [comment]
@WillemJiang The License files has been updated, I have compared with actual distribution libs and updated it as applicable.....
For your reference, the contents of lib folders are 
```
./javax.el-3.0.0.jar
./spring-aspects-4.3.20.RELEASE.jar
./commons-logging-1.2.jar
./spring-boot-starter-1.5.17.RELEASE.jar
./log4j-slf4j-impl-2.7.jar
./spring-jdbc-4.3.20.RELEASE.jar
./spring-boot-autoconfigure-1.5.17.RELEASE.jar
./javax.json-1.0.4.jar
./org.eclipse.persistence.core-2.7.1.jar
./scala-library-2.12.6.jar
./log4j-core-2.7.jar
./slf4j-api-1.7.25.jar
./org.eclipse.persistence.antlr-2.7.1.jar
./config-1.3.3.jar
./fluent-hc-4.5.3.jar
./javax.transaction-api-1.2.jar
./saga-format-0.3.0-SNAPSHOT.jar
./javax.persistence-2.2.0.jar
./spring-boot-1.5.17.RELEASE.jar
./jcl-over-slf4j-1.7.25.jar
./kamon-annotation_2.12-0.6.7.jar
./jackson-annotations-2.9.0.jar
./httpclient-4.5.3.jar
./logback-classic-1.1.11.jar
./spring-beans-4.3.20.RELEASE.jar
./org.eclipse.persistence.jpa.jpql-2.7.1.jar
./commons-codec-1.9.jar
./scala-java8-compat_2.12-0.8.0.jar
./spring-orm-4.3.20.RELEASE.jar
./saga-core-akka-0.3.0-SNAPSHOT.jar
./org.eclipse.persistence.jpa-2.7.1.jar
./snakeyaml-1.17.jar
./jul-to-slf4j-1.7.25.jar
./spring-data-jpa-1.11.16.RELEASE.jar
./transport-httpclient-0.3.0-SNAPSHOT.jar
./spring-data-commons-1.13.16.RELEASE.jar
./spring-context-4.3.20.RELEASE.jar
./aspectjweaver-1.8.10.jar
./spring-core-4.3.20.RELEASE.jar
./log4j-over-slf4j-1.7.25.jar
./akka-actor_2.12-2.5.14.jar
./log4j-api-2.7.jar
./spring-boot-starter-jdbc-1.5.17.RELEASE.jar
./tomcat-jdbc-8.5.34.jar
./saga-core-0.3.0-SNAPSHOT.jar
./jackson-core-2.9.0.jar
./jackson-databind-2.9.0.jar
./kamon-core_2.12-0.6.7.jar
./saga-persistence-jpa-0.3.0-SNAPSHOT.jar
./spring-aop-4.3.20.RELEASE.jar
./HdrHistogram-2.1.9.jar
./httpcore-4.4.6.jar
./disruptor-3.3.7.jar
./org.eclipse.persistence.asm-2.7.1.jar
./tomcat-juli-8.5.34.jar
./spring-expression-4.3.20.RELEASE.jar
./commons-lang3-3.6.jar
./spring-boot-starter-logging-1.5.17.RELEASE.jar
./akka-slf4j_2.12-2.5.14.jar
./spring-tx-4.3.20.RELEASE.jar
./spring-boot-starter-aop-1.5.17.RELEASE.jar
./logback-core-1.1.11.jar


```

**Pull request merged**